documentation/guides/breakpoints-migration.md

```diff
@@ -0,0 +1,751 @@
+}
+```
+
+</details>
+
+#### `when-typography-condensed()`
+
+_Before_
+
+```scss
+@include when-typography-condensed() {
+  padding: 1em;
+}
+```
+
+_After_
+
+```scss
+@media #{$p-breakpoints-md-down} {
+  padding: 1em;
+}
+```
+
+<details>
+<summary>Deprecated Mixin Definition</summary>
+
+```scss
+// Check the Variables section for the value of $typography-condensed
+
+@mixin when-typography-condensed {
+  @include breakpoint-before($typography-condensed) {
+    @content;
+  }
+}
+```
+
+</details>
+
+#### `when-typography-not-condensed()`
+
+_Before_
+
+```scss
+@include when-typography-not-condensed() {
+  padding: 1em;
+}
+```
+
+_After_
+
+```scss
+@media #{$p-breakpoints-md-up} {
+  padding: 1em;
+}
+```
+
+<details>
+<summary>Deprecated Mixin Definition</summary>
+
+```scss
+// Check the Variables section for the value of $typography-condensed
+
+@mixin when-typography-not-condensed {
+  @include breakpoint-after($typography-condensed) {
+    @content;
+  }
+}
+```
+
+</details>
+
+#### `frame-when-nav-displayed()`
+
+_Before_
+
+```scss
+@include frame-when-nav-displayed() {
+  padding: 1em;
+}
+```
+
+_After_
+
+```scss
+@media #{$p-breakpoints-md-up} {
+  padding: 1em;
+}
+```
+
+<details>
+<summary>Deprecated Mixin Definition</summary>
+
+```scss
+@mixin frame-when-nav-displayed() {
+  @include breakpoint-after(48.0625rem) {
+    @content;
+  }
+}
+```
+
+</details>
+
+#### `frame-when-nav-hidden()`
+
+_Before_
+
+```scss
+@include frame-when-nav-hidden() {
+  padding: 1em;
+}
+```
+
+_After_
+
+```scss
+@media #{$p-breakpoints-md-down} {
+  padding: 1em;
+}
+```
+
+<details>
+<summary>Deprecated Mixin Definition</summary>
+
+```scss
+@mixin frame-when-nav-hidden() {
+  @include breakpoint-before(48.0625rem, false) {
+    @content;
+  }
+}
+```
+
+</details>
+
+#### `frame-with-nav-when-not-max-width()`
+
+_Before_
+
+```scss
+@include frame-with-nav-when-not-max-width() {
+  padding: 1em;
+}
+```
+
+_After_
+
+```scss
+@media #{$p-breakpoints-xl-down} {
+  padding: 1em;
+}
+```
+
+<details>
+<summary>Deprecated Mixin Definition</summary>
+
+```scss
+// Check the Variables section for the value of $frame-with-nav-max-width
+
+@mixin frame-with-nav-when-not-max-width() {
+  @include breakpoint-before($frame-with-nav-max-width) {
+    @content;
+  }
+}
+```
+
+</details>
+
+#### `after-topbar-sheet()`
+
+_Before_
+
+```scss
+@include after-topbar-sheet() {
+  padding: 1em;
+}
+```
+
+_After_
+
+```scss
+@media #{$p-breakpoints-sm-up} {
+  padding: 1em;
+}
+```
+
+<details>
+<summary>Deprecated Mixin Definition</summary>
+
+```scss
+@mixin after-topbar-sheet {
+  @include breakpoint-after(450px) {
+    @content;
+  }
+}
+```
+
+</details>
+
+### Adding the functions and mixins to your repo
+
+Any functions or mixins that were being consumed from `???` have been removed. If you wish to continue using them you can add them directly to your repo. All of the removed functions and mixins can found in the following file:
+
+[File Name](#)
```

Comment:
Another thing @lgriffee 

This part was here before, but I didn't know exactly the intention behind it and at the same time it seems important so I didn't remove it hahahaha
But what should I do here? Is the file in question `legacy-polaris-v8-public-api.scss`?